File: srm169/Swimmers.java
public class Swimmers {
	public int[] getSwimTimes(int[] distances, int[] speed, int current) {
		int n = speed.length;
		int[] times = new int[n];
		for (int i = 0; i < times.length; i++) {
			if (distances[i] == 0) {
				times[i] = 0;
				continue;
			} else if ((speed[i] - current) <= 0) {
				times[i] = -1;
				continue;
			} else {
				float tmp = (float)distances[i] / (speed[i] + current) + (float)distances[i] / (speed[i] - current);
				times[i] = (int)tmp;
			}
		}
	return times;
    }
}

File: srm169/swimmers.py
class Swimmers:
	def getSwimTimes(self, distances, speed, current):
		times = [0 for i in range(len(speed))]
		for i in range(len(times)):
			if distances[i] == 0: 
				times[i] = 0
				continue
			if speed[i] - current <= 0: 
				times[i] = -1
				continue
			else:
				times[i] = (float(distances[i]) / (speed[i] + current)) + (float(distances[i]) / (speed[i] - current))
		
		return times